cli/print.go

```diff
@@ -26,6 +27,17 @@ func printf(w io.Writer, format string, a ...interface{}) {
 	fmt.Fprintf(w, format+"\n", a...)
 }
 
+// debugf prints a message prefixed with a bold grey "Debug: ".
+func debugf(c *cli.Context, format string, a ...interface{}) {
```

Comment:
[nit] could this take an `io.Writer` like the other `f`-printing methods? That way, we can decide at call-site whether we want to print to `c.App.Writer` or `c.App.ErrWriter` (can't remember if those are the actual names, but I'm referring to STDOUT vs. STDERR).